src/EnergyPlus/VariableSpeedCoils.cc

```diff
@@ -3987,7 +3987,10 @@ namespace VariableSpeedCoils {
             !state.dataVariableSpeedCoils->MyPlantScanFlag(DXCoilNum)) {
             // for each furnace, do the sizing once.
             ErrorsFound = false;
-            SizeVarSpeedCoil(state, DXCoilNum, ErrorsFound);
+            if (state.dataVariableSpeedCoils->VarSpeedCoil(DXCoilNum).RatedCapCoolTotal > 0.0 ||
+                state.dataVariableSpeedCoils->VarSpeedCoil(DXCoilNum).RatedCapCoolTotal == AutoSize) {
+                SizeVarSpeedCoil(state, DXCoilNum, ErrorsFound);
+            }
```

Comment:
So we won't report to the eio now? Shouldn't we show the user that they have 0 capacity? I think maybe protect the call to CalcCBF would allow sizing to proceed? or add a return in CalcCBF if capacity = 0 to protect all coil models?

Reply:
@rraustad I'll look at modifying that and see what I can come up with.  I think a return at the top of CalcCBF would be a reasonable approach.

Reply:
@rraustad Ok, I reinstated the sizing routine...but it wasn't as simple as skipping the CalcCBF routine.  I had to skip a couple of other things as well to avoid other issues.  I just made another commit that fixes all that and produced the same results as before.  Maybe the slight/couple issues in ci will also be taken care of by this iteration?